moveit_ros/planning/planning_pipeline/src/planning_pipeline.cpp

```diff
@@ -242,57 +176,27 @@ bool planning_pipeline::PlanningPipeline::generatePlan(const planning_scene::Pla
                                                        std::vector<std::size_t>& adapter_added_state_index) const
 {
   // broadcast the request we are about to work on, if needed
-  if (publish_received_requests_)
-    received_request_publisher_->publish(req);
+  if (is_publishing_received_requests_)
+    middleware_handle_->publishReceivedRequest(req);
   adapter_added_state_index.clear();
 
-  if (!planner_instance_)
-  {
-    RCLCPP_ERROR(LOGGER, "No planning plugin loaded. Cannot plan.");
-    return false;
-  }
+  bool isSolved = middleware_handle_->plan(planning_scene, req, res, adapter_added_state_index);
```

Comment:
Variables should be `snake_case` in this code base, so this should be `is_solved`.